sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerConfiguration.java

```diff
@@ -82,17 +83,12 @@ public JwtDecoder jwtDecoder() {
      * bean to override it.
      */
     @Configuration
-    @ConditionalOnMissingBean(WebSecurityConfigurerAdapter.class)
     @EnableWebSecurity
-    public static class DefaultAzureOAuth2ResourceServerWebSecurityConfigurerAdapter extends
-        WebSecurityConfigurerAdapter {
-
+    @ConditionalOnMissingBean(WebSecurityConfigurerAdapter.class)
+    public static class DefaultWebApiSecurityConfigurerAdapter extends AADResourceServerSecurityConfigurerAdapter {
```

Comment:
Echo @chenrujun's comment, we need to make our class name consistent. 
- The class should be named xxxWebSecurityConfigurerAdapter
- Don't mix WebApi and ResourceServer, let's all use ResourceServer here